projector-server/build.gradle

```diff
@@ -78,12 +84,12 @@ if (project.rootProject.file('local.properties').canRead()) {
 }
 
 // relay arguments
-def relayURL=localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_URL"]
-def serverId=localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_SERVER_ID"]
+def relayURL = localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_URL"]
+def serverId = localProperties["ORG_JETBRAINS_PROJECTOR_SERVER_RELAY_SERVER_ID"]
```

Comment:
Please don't change formatting in those lines that you don't edit

Reply:
sorry, I just use `ctrl+alt+L`

Reply:
sorry, I just use `ctrl+alt+L`

Reply:
Well, for some reason we have different code style settings for the project. We should address this later, adding a linter on CI.

Anyway, the recommended commit actions are [the following](https://jetbrains.github.io/projector-client/mkdocs/latest/about/contributing/#submitting-changes) and they reformat only changed lines:

![image](https://user-images.githubusercontent.com/20105593/130446114-6e22842d-091b-425f-8767-d959f911ff59.png)

You can try use only it for now.